ui/editor/index.tsx

```diff
@@ -140,12 +141,13 @@ export default function Editor() {
       onClick={() => {
         editor?.chain().focus().run();
       }}
-      className="relative min-h-[500px] w-full max-w-screen-lg border-stone-200 p-12 px-8 sm:mb-[calc(20vh)] sm:rounded-lg sm:border sm:px-12 sm:shadow-lg"
+      className=" min-h-[500px] w-full max-w-screen-lg border-stone-200 p-12 px-8 sm:mb-[calc(20vh)] sm:rounded-lg sm:border sm:px-12 sm:shadow-lg"
```

Comment:
This breaks the Save Status component's positioning:

![CleanShot 2023-06-25 at 16 08 07](https://github.com/steven-tey/novel/assets/28986134/e9ba65f3-a9e8-4837-b0b7-218f170ca04a)

Reply:
Ohh, I apologise for that. I'm resolving it now.